src/pages/RFOX/components/Unstake/UnstakeInput.tsx

```diff
@@ -388,21 +404,24 @@ export const UnstakeInput: React.FC<UnstakeRouteProps & UnstakeInputProps> = ({
             bg='background.surface.raised.accent'
             borderBottomRadius='xl'
           >
-            <Button
-              size='lg'
-              mx={-2}
-              onClick={handleWarning}
-              colorScheme={Boolean(errors.amountFieldInput) ? 'red' : 'blue'}
+            <ButtonWalletPredicate
+              isValidWallet={Boolean(isChainSupportedByWallet)}
               isDisabled={Boolean(
                 !hasEnteredValue ||
                   !isUnstakeFeesSuccess ||
                   Boolean(errors.amountFieldInput) ||
                   !cooldownPeriod,
               )}
+              size='lg'
+              mx={-2}
+              onClick={handleWarning}
+              colorScheme={Boolean(errors.amountFieldInput) ? 'red' : 'blue'}
               isLoading={isUnstakeFeesLoading}
             >
-              {errors.amountFieldInput?.message || translate('RFOX.unstake')}
-            </Button>
+              {errors.amountFieldInput?.message ||
+                chainNotSupportedByWalletCopy ||
```

Comment:
suggestion (for a follow-up): react-hook-form validation here too

Reply:
Same as https://github.com/shapeshift/web/pull/7270#discussion_r1662986523